package.json

```diff
@@ -0,0 +1,106 @@
+            "require": "./lib/index.js"
+        }
+    },
+    "description": "Install conditional dependencies",
+    "keywords": [
+        "npm",
+        "yarn",
+        "conditional",
+        "dependencies",
+        "package"
+    ],
+    "homepage": "https://github.com/iLib-js/conditionalInstall",
+    "bugs": "https://github.com/iLib-js/conditionalInstall/issues",
+    "email": "marketing@translationcircle.com",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "edwin@translationcircle.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        }
+    ],
+    "files": [
+        "src",
+        "lib",
+        "conditional-install.js",
+        "docs",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:iLib-js/conditionalInstall.git"
+    },
+    "scripts": {
+        "build": "npm-run-all build:prod build:pkg",
+        "build:prod": "grunt babel --mode=prod",
+        "build:dev": "grunt babel --mode=dev",
+        "build:pkg": "echo '{\"type\": \"commonjs\"}' > lib/package.json",
+        "dist": "npm-run-all doc build:prod build:pkg; npm pack",
+        "test": "npm run test:all",
+        "test:cli": "LANG=en_US.UTF8 npm run build:dev ; node --experimental-vm-modules node_modules/.bin/jest --env=node",
+        "test:watch": "LANG=en_US.UTF8 node --experimental-vm-modules node_modules/.bin/jest --watchAll",
+        "test:all": "npm-run-all test:cli",
+        "debug": "npm run build:dev ; node --experimental-vm-modules --inspect-brk node_modules/.bin/jest -i",
+        "debug:web": "LANG=en_US.UTF8 node --experimental-vm-modules node_modules/.bin/karma start --reporters dots",
+        "clean": "git clean -f -d src test; rm -rf lib",
+        "doc": "mkdir -p docs ; jsdoc2md -c jsdoc.json --separators --source src/* -m table > docs/conditionalInstall.md",
+        "doc:html": "jsdoc -c jsdoc.json",
+        "test:install": "node conditional-install.js"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.23.5",
+        "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+        "@babel/plugin-transform-modules-commonjs": "^7.23.3",
+        "@babel/preset-env": "^7.23.5",
+        "@babel/runtime": "^7.23.5",
+        "babel-plugin-add-module-exports": "^1.0.4",
+        "babel-plugin-module-resolver": "^5.0.0",
+        "babel-plugin-transform-import-meta": "^2.2.1",
+        "docdash": "^2.0.2",
+        "grunt": "^1.6.1",
+        "grunt-babel": "^8.0.0",
+        "grunt-cli": "^1.4.3",
+        "grunt-contrib-clean": "^2.0.1",
+        "grunt-contrib-jshint": "^3.2.0",
+        "grunt-contrib-uglify": "^5.2.2",
+        "jsdoc": "^4.0.2",
+        "jsdoc-to-markdown": "^8.0.0",
+        "load-grunt-tasks": "^5.1.0",
```

Comment:
If I npm install for testing this repository, It didn't install `jest`
Shouldn't I add the post install script to this repo's packages.json as well?

Reply:
You have to run the `npm run test:install` script first. I couldn't put it in "postinstall" to run automatically because if you are on earlier versions of node, the "build" script hasn't been run first so the "lib" directory is not available yet. Maybe I should put `"postinstall": "npm-run-all build test:install"` ?

Reply:
Oh wow! I just tried it and it works. Pushing shortly.